docs/extensions/services/accessing-database.md

```diff
@@ -0,0 +1,72 @@
+---
+description: Learn about the ItemsService in Directus and how to utilize them when building extensions.
+contributors: Esther Agbaje
+---
+
+# Accessing Items in the Database
+
+The `ItemsService` allows you to work with and retrieve items data by creating endpoints to items in a collection. It
+provides a way to perform CRUD operations on items in a database.
+
+## Create an Item in a Specific Collection
+
+```js
+  router.post('/items/:collection', async (req, res) => {
+    const schema = await getSchema();
+    const { collection } = req.params;
+
+    const service = new ItemsService(collection, { schema });
+    const data = await service.createOne(req.body);
+
+    res.json(data);
+  });
+```
+
+## Get an Item from a Specific Collection
+
+```js
+router.get('/items/:collection/:id', async (req, res) => {
+    const schema = await getSchema();
+
+    const { collection, id } = req.params;
+    const service = new ItemsService(collection, { schema });
+
+    const data = await service.readOne(id);
+    res.json(data);
+  });
```

Comment:
Fix indentation. Make sure it's consistent.

Reply:
General across the board so I'll stop adding this comment but please do a sweep.